mindjunkies/courses/views.py

```
@@ -39,30 +38,53 @@
     return render(request, "courses/course_list.html", context)
 
 
-class NewCourseView(TemplateView):
-    template_name = "courses/new_course.html"
+from django.views.generic import TemplateView
```

Ruff (0.8.2)

Module level import not at top of file

(E402)